docs/docs/installation.mdx

```diff
@@ -316,3 +323,25 @@ To download a specific version, specify the version number:
 ```bash
 pip3 install rasa==3.0
 ```
+
+## M1 / M2 (Apple Silicon) Limitations
+
+Rasa installations on Apple Silicon use Apple Metal leading to reduced training times
```

Comment:
What is Apple Metal? This might need a clarification.

Reply:
its Apples ML / GPU acceleration, I'll add a link :+1:

```suggestion
Rasa installations on Apple Silicon use [Apple Metal](https://developer.apple.com/metal/) leading to reduced training times
```